src/directives/formly-form.js

```diff
@@ -188,6 +188,9 @@ function formlyForm(formlyUsability, formlyWarn, $parse, formlyConfig, $interpol
             field);
         }
       }
+      if (field.model) {
+        $scope.$watch(() => field.model, onModelOrFormStateChange, true);
```

Comment:
@kentcdodds I just realised, this line might trigger multiple watch handlers if multiple fields use the same model object. I don't know if that is wrong in particular though.
Maybe we should create a set of model objects and create always only a single watcher for each?